spec/requests/user/user_settings_spec.rb

```diff
@@ -122,28 +124,38 @@
         expect(response).to have_http_status(:ok)
       end
 
-      it "shows the 'Remove OAuth' section if a user has multiple enabled identities" do
+      it "shows the description if a user has multiple enabled identities" do
         allow(Authentication::Providers).to receive(:enabled).and_return(Authentication::Providers.available)
         providers = Authentication::Providers.available.first(2)
         allow(user).to receive(:identities).and_return(user.identities.where(provider: providers))
 
         get user_settings_path(tab: "account")
-        expect(response.body).to include(remove_oauth_section)
+        expect(response.body).to include(remove_oauth_description)
+        expect(response.body).not_to include(remove_oauth_instructions)
       end
 
-      it "hides the 'Remove OAuth' section if a user has one enabled identity" do
+      it "shows instructions on how to remove a identity if a user has one enabled identity" do
```

Comment:
nitpicking grammar here and on line 147 below: I think we ought to use`shows instructions how to remove an identity` rather than `a identity`

Reply:
Thank you, I fixed it.